documentation/samplefiles/readme.python.md

```diff
@@ -4,18 +4,35 @@ These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
 ```yaml $(python)
+python-mode: create
 python:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   payload-flattening-threshold: 2
-  namespace: Microsoft.YourServiceName
-  package-name: YourServiceName
-  package-version: 2019-12-01
+  namespace: [[ResourceProviderName]]
+  package-name: azure-mgmt-[[ServiceName]]
+  package-version: [[Version]]
```

Comment:
@changlong-liu could you help check the sample python readme file?

Reply:
CC to @jsntcy 

Hi @JackTn, @leni-msft ,
1. Since Track2 is GAed last week and all new service tend to use track2, so we can remove track1 template in this template.
2. several issues found in this sample, suggest the yaml part can be replaced as below:

~~~
``` yaml $(track2)
azure-arm: true
license-header: MICROSOFT_MIT_NO_VERSION
package-name: azure-mgmt-[[ServiceName]]
no-namespace-folders: true
package-version: 1.0.0b1
```

``` yaml $(python-mode) == 'update'
no-namespace-folders: true
output-folder: $(python-sdks-folder)/[[ServiceName]]/azure-mgmt-[[ServiceName]]/azure/mgmt/[[ServiceName]]
```
``` yaml $(python-mode) == 'create'
basic-setup-py: true
output-folder: $(python-sdks-folder)/[[ServiceName]]/azure-mgmt-[[ServiceName]]
```
~~~

Reply:
it has been changed, pls confirm it @changlong-liu

Reply:
Thanks! looks good to me now.